examples/platform_view/lib/main.dart

```diff
@@ -47,6 +47,17 @@ class _MyHomePageState extends State<MyHomePage> {
     });
   }
 
+  Text _getButtonText() {
+    if (Platform.isAndroid) {
```

Comment:
nit: use TargetPlatform and a switch case so that we know what is unimplemented

```dart
switch(defaultTargetPlatform) {
      case TargetPlatform.android:
        return const Text('Continue in Android view');
      case TargetPlatform.iOS:
        return const Text('Continue in iOS view');
      case TargetPlatform.windows:
        return const Text('Continue in Windows view');
      case TargetPlatform.fuchsia:
      case TargetPlatform.macOS:
      case TargetPlatform.linux:
        throw UnimplementedError();
    }
```
You can choose to ignore this.